tools/rosbag/test/test_roundtrip.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import rosbag
+import rospy
+
+class TestRoundTrip(unittest.TestCase):
+    def _write_simple_bag(self):
+        from std_msgs.msg import Int32, String
+
+        with rosbag.Bag('/tmp/test.bag', 'w') as bag:
```

Comment:
Please use a bag name which is kind of unique to this test. Otherwise it might collide with other tests being run in parallel.